server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CommonCobolLexer.g4

```diff
@@ -0,0 +1,964 @@
+WAIT : W A I T;
+WAITCICS : W A I T C I C S;
+WEB : W E B;
+WPMEDIA : W P M E D I A;
+WPMEDIA1 : W P M E D I A '1';
+WPMEDIA2 : W P M E D I A '2';
+WPMEDIA3 : W P M E D I A '3';
+WPMEDIA4 : W P M E D I A '4';
+WRAP : W R A P;
+WRITE : W R I T E;
+WRITEQ : W R I T E Q;
+WSACONTEXT : W S A C O N T E X T;
+WSAEPR : W S A E P R;
+XCTL : X C T L;
+XMLCONTAINER : X M L C O N T A I N E R;
+XMLTODATA : X M L T O D A T A;
+XMLTRANSFORM : X M L T R A N S F O R M;
+XRBA : X R B A;
+YEAR : Y E A R;
+YYDDD : Y Y D D D;
+YYDDMM : Y Y D D M M;
+YYMMDD : Y Y M M D D;
+YYYYDDD : Y Y Y Y D D D;
+YYYYDDMM : Y Y Y Y D D M M;
+YYYYMMDD : Y Y Y Y M M D D;
+ZERO_DIGITAL : Z E R O '_' D I G I T A L;
+
+// symbols
+AMPCHAR : '&';
+ASTERISKCHAR : '*';
+DOUBLEASTERISKCHAR : '**';
+COLONCHAR : ':';
+COMMA_EOF : ',' EOF {!sqlFlag}? ->skip;
+COMMA_LB : ',' ('\r' | '\n' | '\f' | '\t' | ' ')+ {!sqlFlag}? ->skip;
+COMMACHAR : ',';
+COMMENTTAG : '*>';
+COMMENTENTRYTAG : '*>CE';
+DOLLARCHAR : '$';
+DOUBLEQUOTE : '"';
+// period full stopPosition
+DOT_FS : '.' ('\r' | '\n' | '\f' | '\t' | ' ')+ | '.' EOF;
+DOT : '.';
+EQUALCHAR : '=';
+LESSTHANCHAR : '<';
+LESSTHANOREQUAL : '<=';
+LPARENCHAR : '(';
+MINUSCHAR : '-';
+MORETHANCHAR : '>';
+MORETHANOREQUAL : '>=';
+NOTEQUALCHAR : '<>';
+PLUSCHAR : '+';
+SEMICOLON_FS : ';' ('\r' | '\n' | '\f' | '\t' | ' ')+ | ';' EOF;
+SINGLEQUOTE : '\'';
+RPARENCHAR : ')';
+SLASHCHAR : '/';
+SQLLINECOMMENTCHAR: '--';
+
+LEVEL_NUMBER: ([1-9])|([0][1-9])|([1234][0-9]);
+LEVEL_NUMBER_66 : '66';
+LEVEL_NUMBER_77 : '77';
+LEVEL_NUMBER_88 : '88';
+NUMBER_1 : '1';
+NUMBER_2 : '2';
+NUMBER_4 : '4';
+NUMBER_5 : '5';
+NUMBER_6 : '6';
+NUMBER_8 : '8';
+NUMBER_10 : '10';
+NUMBER_12: '12';
+NUMBER_14 : '14';
+NUMBER_15 : '15';
+NUMBER_16 : '16';
+NUMBER_20 : '20';
+NUMBER_30 : '30';
+NUMBER_31 : '31';
+NUMBER_33 : '33';
+NUMBER_34 : '34';
+NUMBER_64 : '64';
+NUMBER_100 : '100';
+NUMBER_256 : '256';
+NUMBER_1200 : '1200';
+NUMBER_1208 : '1208';
+INTEGER_MAX : '2147483647';
+
+INTEGERLITERAL : (PLUSCHAR | MINUSCHAR)? DIGIT+ | LEVEL_NUMBER | LEVEL_NUMBER_66 | LEVEL_NUMBER_77 | LEVEL_NUMBER_88 |
+                  NUMBER_1 | NUMBER_2 | NUMBER_4 | NUMBER_5 | NUMBER_6 | NUMBER_10 | NUMBER_12 | NUMBER_14 | NUMBER_15 |
+                  NUMBER_16 | NUMBER_20 | NUMBER_30 NUMBER_31 | NUMBER_33 | NUMBER_34 | NUMBER_64 | NUMBER_100 | NUMBER_256 |
+                  NUMBER_1200 | NUMBER_1208 | INTEGER_MAX ;
+
+// DECIMAL_CONST : DIGIT DIGIT DIGIT DIGIT DOT DIGIT {sqlFlag}? ;
+SINGLEDIGITLITERAL : DIGIT {sqlFlag}? ;
+
+NUMERICLITERAL : (PLUSCHAR | MINUSCHAR)? DIGIT* (DOT | COMMACHAR) DIGIT+ (('e' | 'E') (PLUSCHAR | MINUSCHAR)? DIGIT+)?;
+
+NONNUMERICLITERAL : UNTRMSTRINGLITERAL | STRINGLITERAL | DBCSLITERAL | HEXNUMBER | NULLTERMINATED;
+
+//TXTLITERAL : STRINGLITERAL | IDENTIFIER;
+CHAR_STRING_CONSTANT : HEXNUMBER | STRINGLITERAL;
+
+IDENTIFIER : ([a-zA-Z0-9]+ ([-_]+ [a-zA-Z0-9]+)*);
+FILENAME : IDENTIFIER+ '.' IDENTIFIER+; // TODO check , if this is OK?
```

Comment:
This is for consistency with the preprocessor lexer to build mapping.
I don't see a token for `INVALID_IDENTIFIER`

Reply:
added